### DISCUSSION TITLE
be启动脚本报错

### DISCUSSION THREAD

**Discussion by ASanlittleboss** [apache/doris#16511] [A - General / Q&A]
执行./start_be.sh --daemon 报错
./start_be.sh: line 242: ***/lib/doris_be: cannot execute binary file

**Comment by frankmeten** [comment]
Mac : 
./bin/start_be.sh: line 324: /Users/frank/Downloads/software/apache-doris-2.0.0-bin-x64/be/lib/doris_be: cannot execute binary file

$ sysctl -a | grep brand
machdep.cpu.brand_string: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
machdep.cpu.brand: 0

$ sysctl -a | grep avx2
hw.optional.avx2_0: 1
